README.md

```diff
@@ -19,57 +18,42 @@ __This code is completely unsupported. Use at your own risk.__
 
 1. [Emacs](https://www.gnu.org/software/emacs/)
 1. [Mongo C Driver](http://mongoc.org/)
-2. [lib-ffi](https://sourceware.org/libffi/)
-3. [elisp-ffi](https://github.com/skeeto/elisp-ffi)
 
 See below for full installation instructions.
 
 ## Installation
 
 Below targets running on OS X with [homebrew](https://brew.sh/):
 
-Command overview:
-
-    mkdir mongo-mode-install
-    cd mongo-mode-install
-    MONGO_MODE_HOME="$PWD"
+### Build the Mongo C Driver
 
-    # Install Mongo C Drivers
-    brew install pkgconfig
+    brew install pkgconfig cmake
     mkdir mongoc
     pushd mongoc
-        curl -LO https://github.com/mongodb/mongo-c-driver/releases/download/1.8.1/mongo-c-driver-1.8.1.tar.gz
-        tar xzf mongo-c-driver-1.8.1.tar.gz
-        cd mongo-c-driver-1.8.1
-        ./configure --prefix="$MONGO_MODE_HOME/lib"
-        make
-        make install
-        cd ..
+        curl -LO https://github.com/mongodb/mongo-c-driver/releases/download/1.11.0/mongo-c-driver-1.11.0.tar.gz
+        tar xzf mongo-c-driver-1.11.0.tar.gz
```

Comment:
Is this specific version of the c driver required, or will it work with whatever the user has installed?

Reply:
Probably not -- but I know this one works :)